coverage/lcov-report/data/compareInfo.ts.html

```diff
@@ -0,0 +1,322 @@
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">/* eslint-disable max-len */
+&nbsp;
+export default {
+  aboutInfo: [
+    {
+      title: 'Calico Open Source',
+      link: '/calico/latest/about/',
+&nbsp;
+      description: [
+        'The base product that comprises both Calico Enterprise and Calico Cloud. It provides the core networking and network policy features.',
+      ],
+      linkDescription: 'Learn more about Calico Open Source',
+      image: 'img/calico-open-source.svg',
+    },
+    {
+      title: 'Calico Enterprise',
+      link: '/calico-enterprise/latest/about-calico-enterprise',
+&nbsp;
+      description: [
+        'Includes Calico Open Source core networking and network policy, but adds advanced features for networking, network policy, visibility and troubleshooting, threat defense, and compliance reports. ',
+      ],
+      linkDescription: 'Learn more about Enterprise',
+      image: 'img/calico-enterprise.drawio.svg',
+    },
+    {
+      title: 'Calico Cloud',
+&nbsp;
+      link: 'calico-cloud',
+      description: [
+        'The SaaS version of Calico Enterprise. It adds Image Assurance to scan and detect vulnerabilities in images, and container threat defense to detect malware. It also adds onboarding tutorials, and eliminates the cost to manage Elasticsearch logs and storage that comes with Calico Enterprise.',
+      ],
+      linkDescription: 'Learn more about Calico Cloud',
+      image: 'img/calico-cloud-features.svg',
+    },
+  ],
+  bestFit: [
+    {
+      title: 'Calico Open Source',
+      costSupport: ['Free, self-managed'],
+      target: 'Users',
+      image: 'img/calico-logo.webp',
+&nbsp;
+      bestFit: [' who want best-in-class networking and network policy capabilities.'],
+    },
+    {
+      title: 'Calico Enterprise',
+      costSupport: ['Paid, self-managed'],
+      image: '/img/calico-cloud-logo.webp',
+&nbsp;
+      target: 'Enterprise teams',
+&nbsp;
+      bestFit: [
```

Comment:
' - Who need full control to customize their networking security deployment to meet regulatory and',
        ' compliance requirements for Kubernetes at scale.',
        ' - Teams that require Tigera Customer Support best practices to get from day zero to production,',
        ' custom training and workshops, and Solution Architects to customize solutions.',